spec/models/converters/base_converter_spec.rb

```diff
@@ -14,15 +14,16 @@
     end
   end
 
+  it 'does not convert strings containing more than forbidden words to nil' do
```

Comment:
The spec description looks like it missing a word..."containing more than ??? forbidden words"